linear_algebra/src/python-polynom-for-points.py

```diff
@@ -0,0 +1,103 @@
+    
+from decimal import *
+getcontext().prec=28
+
+
+def points_to_polynomial(coordinates):
+    #the list (coordinates) is two dimensional: [[x, y],[x, y],...]
+
+    if __name__ == "__main__" :
```

Comment:
Remove this line here and put it down at the bottom of the script OUTSIDE of __points_to_polynomial()__.